sdk/Lusid.Sdk.Tests/Utilities/InstrumentExamples.cs

```diff
@@ -420,11 +420,11 @@ internal static InterestRateSwaption CreateExampleInterestRateSwaptionWithNamedC
             );
         }
 
-        internal static EquitySwap CreateExampleEquitySwap()
+        internal static EquitySwap CreateExampleEquitySwap(bool multiCoupon = false)
```

Comment:
Consider adding a method comment that describes the purpose of the multiCoupon param